src/extension/monitor/core/src/test/java/org/geoserver/monitor/MonitorFilterTest.java

```diff
@@ -271,6 +271,20 @@ public void testGetStatistics() throws IOException, ServletException {
         assertEquals(data.getLabellingProcessingTime().longValue(), statistics.getLabellingTime());
     }
 
+    @Test
+    public void testDisableReverseDNSProcessor() throws Exception {
```

Comment:
Good, can we also have an assertion somewhere that checks the remote host is there, when the post processor is not disabled?

Reply:
understood, will add